docs/zh/latest/plugins/openwhisk.md

```diff
@@ -0,0 +1,136 @@
+---
+title: openwhisk
+keywords:
+  - APISIX
+  - Plugin
+  - OpenWhisk
+  - openwhisk
+description: 本文介绍了关于 Apache APISIX `openwhisk` 插件的基本信息及使用方法。
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`openwhisk` 插件用于将开源的分布式无服务器平台 [Apache OpenWhisk](https://openwhisk.apache.org) 作为动态上游集成至 APISIX。
+
+启用 `openwhisk` 插件后，该插件会终止对已配置 URI 的请求，并代表客户端向 OpenWhisk 的 API Host 端点发起一个新的请求，然后 `openwhisk` 插件会将响应信息返回至客户端。
```

Comment:
Refer to the [English version](https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/openwhisk.md): `This Plugin can be configured on a Route and requests will be send to the configured OpenWhish API endpoint as the upstream.`

👍Good translation.